arithmetization/src/main/java/net/consensys/linea/plugins/rpc/Requests.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Consensys Software Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package net.consensys.linea.plugins.rpc;
+
+import java.security.InvalidParameterException;
+
+import org.hyperledger.besu.plugin.BesuContext;
+import org.hyperledger.besu.plugin.services.TraceService;
+
+public class Requests {
+
+  /**
+   * Initialize the TraceService.
+   *
+   * @return the initialized TraceService.
+   */
+  public static TraceService getTraceService(final BesuContext besuContext) {
```

Comment:
why not making a method that gets the class as parameter to make it more generic ?
Something along the lines :
public static <T> T getTraceService(final BesuContext besuContext, Class<T> className)
?

Reply:
Sure I can do that.